composeApp/src/commonMain/kotlin/sp/bvantur/tasky/register/presentation/RegisterViewState.kt

```diff
@@ -9,4 +9,6 @@ data class RegisterViewState(
     val isNameError: Boolean = false,
     val isEmailError: Boolean = false,
     val isPasswordError: Boolean = false,
+    val showErrorDialog: Boolean = false,
```

Comment:
How do you usually handle presenting dialogs in compose? In the legacy toolkit with Fragments and XML layouts I always treated Dialogs as one-time events and never included them into a ViewState. With Compose I am sensing this should be done through ViewState. And also than hiding of dialog is done like I did with an additional call to ViewModel after some user interaction with the dialog?

Reply:
Yes, I do it exactly like you